src/box/lua/upgrade.lua

```diff
@@ -2221,6 +2221,7 @@ local downgrade_versions = {
     "3.1.1",
     "3.1.2",
     "3.2.0",
+    "3.2.1",
```

Comment:
Should we also add 2.11.5?

Reply:
I see, it is in the next commit. OK.

Reply:
We do in the next patch. I'm not sure which option would be less confusing. I chose to add "3.2.1" in the commit mentioning "cleanup 3.2.1 changelogs", which seems logical. And I added "2.11.5" in the next commit.

Reply:
I don't mind. Just read it from the beginning and asked instantly :)